pysam/libcalignmentfile.pyx

```diff
@@ -787,7 +791,8 @@ cdef class AlignmentFile(HTSFile):
               referencenames=None,
               referencelengths=None,
               duplicate_filehandle=True,
-              ignore_truncation=False):
+              ignore_truncation=False,
+              format_options=[]):
```

Comment:
Why is it better to use `[]` than `None`?  It is generally bad practice to use mutable values as default values and I didn't see any problems with using `None` as before.